src/stdlib.rs

```diff
@@ -46,6 +47,6 @@ impl<F: Field + From<u64>> IsZero<F> {
 
 impl<F: Field + Eq + Hash> IsZero<F> {
     pub fn wg<WGC>(&self, ctx: &mut StepInstance<F>, value: F) {
```

Comment:
Probably missed this from some other PR but why do we need a <WGC> generic parameter here that's unused?

Reply:
it seems this is unrelated, please create an issue